platform/ios/src/MGLMapView.mm

```diff
@@ -729,11 +735,12 @@ - (void)dealloc
     [_locationManager stopUpdatingLocation];
     [_locationManager stopUpdatingHeading];
     _locationManager.delegate = nil;
-
+    MGLLogInfo(@"Deallocating MGLMapView.");
```

Comment:
"Deallocated" instead?

Reply:
Info logs actions, I prefer the present participle use.